src/auth/ducks/types.ts

```diff
@@ -29,6 +29,46 @@ export interface SignupRequest {
   readonly password: string;
   readonly firstName: string;
   readonly lastName: string;
+  readonly phoneNumber: string;
+  readonly location: {
+    readonly address: string;
+    readonly city: string;
+    readonly state: string;
+    readonly zipCode: string;
+  };
+  readonly photoRelease: boolean;
+}
+export interface SetContactsRequest {
+  mainContact: AdultContact;
+  additionalContacts: AdditionalContact[];
+  children: ChildContact[];
+}
+
+interface Contact {
+  readonly firstName: string;
+  readonly lastName: string;
+  readonly dateOfBirth: string; // YYYY-MM-DD
+  readonly phoneNumber: string;
+  readonly pronouns: string;
+  readonly allergies: string | null;
+  readonly diagnosis: string | null;
+  readonly medications: string | null;
+  readonly notes: string | null;
+  readonly profilePicture: string | null;
+  readonly referrer: string | null;
+}
+
+interface AdultContact extends Contact {
+  readonly email: string;
+}
+
+interface AdditionalContact extends AdultContact {
+  readonly shouldSendEmails: boolean;
+}
+
+interface ChildContact extends Contact {
+  readonly school: string;
+  readonly schoolYear: string;
 }
```

Comment:
These can go in `src/containers/setContacts/types.ts`, that way the sections are not interdependent.